pkg/patch/patch.go

```diff
@@ -231,13 +231,24 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
 		return err
 	})
 
+	var c console.Console
+	if cn, err := console.ConsoleFromFile(os.Stderr); err == nil {
+		c = cn
+	}
+
+	mode := progressui.AutoMode
+	if log.GetLevel() >= log.DebugLevel {
+		mode = progressui.PlainMode
+	}
+
+	display, err := progressui.NewDisplay(c, mode, progressui.WithPhase(c.Name()))
```

Comment:
Not sure `c.Name()` is the right thing here.
Can probably leave it without a phase set.